userguide/tn_catalog.rst

```diff
@@ -0,0 +1,63 @@
+.. _Product Catalog:
+
+Appendix B: %brand% Product Catalog
+=============================================
+
+iXsystems offers many storage products that are designed to fully use
+the many features of %brand%. These products have enterprise features
+like High Availability (HA), high-efficiency storage optimization, and
+fast networking speeds for data transfer. Scalability and modularity are
+also featured.
+
+Each product makes full use of %brand% and have a variety of features at
+different price points. To see hardware specifications or request a
+quote, visit the
+`iXsystems website <https://www.iXsystems.com/truenas>`__ or contact an
+iX representative at :literal:`sales@ixsystems.com`.
+
+
+X-Series
+--------
+
+.. figure:: images/truenas/x-series.png
+
+
+The X-Series is designed to provide reliabile storage at an affordable
+price for small and medium sized IT environments.
+
+
+M-Series
+--------
+
+.. figure:: images/truenas/m40.png
+
+The M-Series combines robust features like high speed networking,
+large volume system memory, and data storage that scales up to ten
+petabytes with the power of %brand% to meet the challenges of heavy
+storage workloads.
+
+
+Expansion Shelves
+-----------------
+
+Expansion shelves are an opportunity to create a %brand% storage array
+with the flexibility to adapt to the evolving challenges of an IT
+environment. Each expansion product supports an additional number of
```

Comment:
Last sentence is not necessary, each expansion shelf intro should tell the number of drives it supports.  And it's not always guaranteed that the number is the number of drives the shelf holds.